configure.ac

```diff
@@ -768,6 +768,7 @@ case "${host}" in
   *-*-nto-qnx)
 	abi="elf"
   force_tls="0"
+  AC_DEFINE([JEMALLOC_HAS_ALLOCA_H])
```

Comment:
Is this necessary? It seems like this should get autodetected correctly as is, assuming alloca.h is in fact correctly detected.

Reply:
It's not detected somehow. I was following other cases around this block, e.g. *-*-linux*) and *-*-linux-android*) where this is set manually.